x-pack/platform/plugins/shared/onechat/public/application/context/tools_table_provider.tsx

```diff
@@ -34,14 +27,34 @@ export const ToolsTableActionsContext = createContext<ToolsActionsContextType |
 );
 
 export const ToolsTableProvider = ({ children }: { children: React.ReactNode }) => {
+  const { navigateToOnechatUrl } = useNavigation();
+
+  const createTool = useCallback(() => {
+    navigateToOnechatUrl(appPaths.tools.new);
+  }, [navigateToOnechatUrl]);
+
+  const editTool = useCallback(
```

Comment:
rn we are using dynamic onClick navigation  in the table for edit / new. this is not a good practice form a18y perspective, we should use hrefs for links (see agent table you can 
```
href={createOnechatUrl(appPaths.tools.new)}
```